web/src/components/Upload.vue

```diff
@@ -69,19 +71,35 @@ export default {
   },
   methods: {
     async onFileChange(targetFiles) {
-      this.pendingFiles = this.pendingFiles.concat([...targetFiles].map(file => ({
-        file,
-        status: 'pending',
-        progress: {},
-        meta: {},
-      })));
-
       const excelMimeTypes = [
         'application/vnd.ms-excel',
         'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet',
       ];
 
       const isExcelFile = file => excelMimeTypes.includes(file.type) || file.name.match(/\.xlsx?$/i);
```

Comment:
This regex doesn't match the accepted types for the dropzone.  `.xlsx?` will match `.xls` too.